manticore/models/system.py

```diff
@@ -0,0 +1,22 @@
+class OSException(Exception):
```

Comment:
what is this file for? models/ is for function models, so should it be somewhere else?